tests/graph/test_centrality.py

```diff
@@ -230,3 +230,90 @@ def test_custom_graph_unnormalized(self):
         expected = {0: 9, 1: 9, 2: 12, 3: 15, 4: 11, 5: 14, 6: 10, 7: 13, 8: 9, 9: 9}
         for k, v in centrality.items():
             self.assertAlmostEqual(v, expected[k])
+
+
+class TestGraphDegreeCentrality(unittest.TestCase):
+    def setUp(self):
+        self.graph = rustworkx.PyGraph()
+        self.a = self.graph.add_node("A")
+        self.b = self.graph.add_node("B")
+        self.c = self.graph.add_node("C")
+        self.d = self.graph.add_node("D")
+        edge_list = [
+            (self.a, self.b, 1),
+            (self.b, self.c, 1),
+            (self.c, self.d, 1),
+        ]
+        self.graph.add_edges_from(edge_list)
+
+    def test_degree_centrality(self):
+        centrality = rustworkx.graph_degree_centrality(self.graph)
+        expected = {
+            0: 1 / 3,  # Node A has 1 edge, normalized by (n-1) = 3
+            1: 2 / 3,  # Node B has 2 edges
+            2: 2 / 3,  # Node C has 2 edges
+            3: 1 / 3,  # Node D has 1 edge
+        }
+        self.assertEqual(expected, centrality)
+
+    def test_degree_centrality_complete_graph(self):
+        graph = rustworkx.generators.complete_graph(5)
+        centrality = rustworkx.graph_degree_centrality(graph)
+        expected = {0: 1.0, 1: 1.0, 2: 1.0, 3: 1.0, 4: 1.0}
+        self.assertEqual(expected, centrality)
+
+    def test_degree_centrality_star_graph(self):
+        graph = rustworkx.generators.star_graph(5)
+        centrality = rustworkx.graph_degree_centrality(graph)
+        expected = {0: 1.0, 1: 0.25, 2: 0.25, 3: 0.25, 4: 0.25}
+        self.assertEqual(expected, centrality)
+
+    def test_degree_centrality_empty_graph(self):
+        graph = rustworkx.PyGraph()
+        centrality = rustworkx.graph_degree_centrality(graph)
+        expected = {}
+        self.assertEqual(expected, centrality)
+
+class TestDiGraphDegreeCentrality(unittest.TestCase):
+    def setUp(self):
+        self.digraph = rustworkx.PyDiGraph()
+        self.a = self.digraph.add_node("A")
+        self.b = self.digraph.add_node("B")
+        self.c = self.digraph.add_node("C")
+        self.d = self.digraph.add_node("D")
+        edge_list = [
+            (self.a, self.b, 1),
+            (self.b, self.c, 1),
+            (self.c, self.d, 1),
+        ]
+        self.digraph.add_edges_from(edge_list)
+
+    def test_digraph_degree_centrality(self):
+        centrality = rustworkx.digraph_degree_centrality(self.digraph)
+        expected = {
+            0: 1 / 3,  # Node A has 1 outgoing edge
+            1: 2 / 3,  # Node B has 1 incoming and 1 outgoing edge
+            2: 2 / 3,  # Node C has 1 incoming and 1 outgoing edge
+            3: 1 / 3,  # Node D has 1 incoming edge
+        }
+        self.assertEqual(expected, centrality)
+
+    def test_digraph_degree_centrality_with_direction(self):
```

Comment:
Minor organization detail. This should put the digraph tests in tests/digraph/test_centrality.py with the other directed graph tests